include/mc_tasks/lipm_stabilizer/StabilizerTask.h

```diff
@@ -95,6 +95,30 @@ struct MC_TASKS_DLLAPI StabilizerTask : public MetaTask
                  unsigned int robotIndex,
                  double dt);
 
+  /** Set a name for the task
+   *
+   * This name will be used to identify the task in logs, GUI...
+   *
+   * The name should be set before being added to the solver.
+   *
+   */
+  virtual inline void name(const std::string & name) override
```

Comment:
`override` only

Reply:
Why we do not need `inline` here? You gave me the comment about `inline` before  https://github.com/jrl-umi3218/mc_rtc/pull/95#discussion_r595662632

Reply:
Ah sorry, I meant no `virtual`, `inline` is fine :D